api/views.py

```diff
@@ -84,6 +102,16 @@ def get_delivery_log(request, id):
         log = user.delivery_logs.get(id=id)
     except DeliveryLog.DoesNotExist:
         return Response(status=404, data={"error": f" Message ID: {id} does not exist"})
```

Comment:
If the `404` status reads would be useful to audit log then audit logging them here could be added and the `create_api_commit_message_from_request` e.g. parametrized with `status` parameter to make the failed fetching loggable using it.

Reply:
I would not like to include them. I don't know what other are thinking about it. I think those are just spam and if the action does not have an actor (or has an anonymous actor), it should not be in the scope of audit log. Access log is already collected by the server instance.

Reply:
It is not good to use a postgres database to write "access logs".